ipc/uapi_tamago.go

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: MIT
+ *
+ * Copyright (C) 2017-2023 WireGuard LLC. All Rights Reserved.
+ */
+
+//go:build tamago
```

Comment:
I thought this package already had a shared stub file for all platforms that needed these fake numbers? (Like rwcancel, etc)

Is that easy rather than adding a new file?

Reply:
Nope, uapi is structured differently (see uapi_js.go) for instance. I think I am following the existing code base here.